src/Header/MobileMenu.tsx

```diff
@@ -0,0 +1,47 @@
+import classnames from 'classnames';
+import React from 'react';
+import {AppLinks} from './AppLinks';
+import {MainMenu} from './MainMenu';
+import {SecondaryMenu} from './SecondaryMenu';
+import {UserAvatar} from './UserAvatar';
+
+export interface MobileMenuProps {
+  className?: string;
+  userName?: string;
+  onCloseClick: (e: React.MouseEvent<HTMLAnchorElement>) => void;
+}
+
+export const MobileMenu: React.SFC<MobileMenuProps> = ({
+  className,
+  userName,
+  onCloseClick,
+  ...attributes
+}) => {
+  return (
+    <div {...attributes} className={classnames('mobile-menu', className)}>
+      <div className="mobile-menu__header">
+        <a href="#" onClick={onCloseClick} className="mobile-menu__close-btn" />
+        {userName ? (
+          <UserAvatar userName={userName} />
+        ) : (
+          <div className="mobile-menu__get-started">
+            <a href="http://lykke.com/site/signin" className="btn btn-primary">
```

Comment:
I think this part can be saved to separate component because you have same code in other file